Lib/threading.py

```diff
@@ -980,15 +987,24 @@ def start(self):
         if self._started.is_set():
             raise RuntimeError("threads can only be started once")
 
+        self._join_lock = _allocate_lock()
+
         with _active_limbo_lock:
             _limbo[self] = self
         try:
-            _start_new_thread(self._bootstrap, ())
+            # Start joinable thread
+            _start_new_thread(self._bootstrap, (), {}, True)
```

Comment:
It isn't critical of course, but a keyword argument for the last arg might help readability:

```suggestion
            _start_new_thread(self._bootstrap, (), {}, joinable=True)
```

Reply:
Yes, I might bite the bullet and switch this API to `PyArg_ParseTupleAndKeywords` finally.

Reply:
It's up to you.  I don't presume to ask you to do a bunch of busy work for what is essentially an internal function.  It would be nice though. 😄

Reply:
I undid this in favor of a new `start_joinable_thread` function.